aim/web/ui_v2/src/pages/Metrics/Metrics.scss

```diff
@@ -31,40 +29,18 @@
     justify-content: center;
   }
 
-  .Metrics__resize {
-    cursor: row-resize;
-    justify-content: center;
-    display: flex;
-    align-items: center;
-    height: 0.5rem;
-    border-top: 0.125rem solid tint( $primary-color, 90);
-    border-bottom: 0.125rem solid  tint( $primary-color, 90);
-    background-color: tint( $primary-color, 90);
-    transition: all .15s ease-out;
-    transition-delay: .3s;
-    i {
-      transition: all .15s ease-out;
-      transition-delay: .3s;
-    }
-    &:hover, &.resizing {
-      background-color: tint( $primary-color, 25);
-      i {
-        opacity: 0;
-      }
-    }
-  }
 
   .Metrics__chart__container {
-    -webkit-box-flex: 0.5;
-    -ms-flex: 0.5 1 0;
-    flex: 0.5 1 0;
     min-height: 372px;
+    &__fullHeight {
+      flex: 1 100% !important;
```

Comment:
Is this necessary to write '!important' here ? Lets write without '!important' if we can

Reply:
Yes it's necessary, because it should override inline style.